dynawo/sources/Common/Dictionaries/DYNTimeline_en_GB.dic

```diff
@@ -40,6 +40,10 @@ PhaseShifterBelowMin      =             Phase-shifter : below minimum allowed va
 PhaseShifterBelowMax      =             Phase-shifter : below maximum allowed value
 PhaseShifterBelowStop     =             Phase-shifter : back to acceptable value
 PhaseShifterWithinInterval=             Phase-shifter : back to acceptable value
+// --> Phase-shifter Blocking
+PhaseShifterBlockingIActing =           Phase-shifter : order to block
+PhaseShifterBlockingIArming =           Phase-shifter : block order arming by over-current constraint
+PhaseShifterBlockingIDisarming =        Phase-shifter : block disarming as current is back to acceptable value
```

Comment:
I think the message should be "block order disarming" because it is the block order that is disarmed because if the current is back to acceptable value after the delay for action the phase shifter remains blocked, isn't it?

Reply:
done